server/controllers/thoughtController.js

```diff
@@ -0,0 +1,127 @@
+const User = require("../model/User");
+const Thought = require("../model/Thought");
+
+const addThought = async (req, res) => {
+    try {
+        const thought = await Thought.create(req.body);
```

Comment:
Consider adding additional validation here e.g. only create the thought if the request contains all of the required fields from the Thought model.
This will prevent your app from breaking if people send requests with incorrect data :-)